tests/test_archimedes.py

```diff
@@ -355,20 +355,20 @@ def test_import_from_disk_no_id_no_title(self):
             archimedes.import_from_disk(VISUALIZATION)
 
     def test_import_from_disk_find_index_pattern(self):
-            """Test whether an error is thrown when importing an index pattern with the flag `find` enabled"""
+        """Test whether an error is thrown when importing an index pattern with the flag `find` enabled"""
```

Comment:
I don't think these tests are related to this PR.

Reply:
yes, you are right, it was a fix in the indentation of this test and `test_import_from_disk_not_found`. I'm going to put it in another PR